edward/inferences.py

```diff
@@ -208,6 +214,10 @@ def initialize(self, n_iter=1000, n_data=None, n_print=100,
         init = tf.initialize_all_variables()
         init.run()
 
+        # Start input enqueue threads.
```

Comment:
The queue runners are necessary for batch training if the data fits in memory (we use `tf.train.slice_input_producer` and `tf.train.batch`, which in turn uses queues). And in general for distributed training when using data readers. 

So it sounds like we should build a method for catching ctrl-c/session interrupts?